extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -268,4 +271,41 @@ export class AmpStoryConsent extends AMP.BaseElement {
         storyConsent.vendors && isArray(storyConsent.vendors),
         `${TAG}: story-consent requires an array of vendors`);
   }
+
+  /**
+   * Sets the accept button font color to either white or black, depending on
+   * the publisher custom background color.
+   * @private
+   */
+  setAcceptButtonFontColor_() {
```

Comment:
Feels like this should be more like a util (or maybe two: the rgb helper and the relative luminance helper) that we import into `amp-story-consent`?  This is not consent-specific 😄  There are also other features we've brainstormed that might want to use this

Reply:
Done